lib/src/commit.rs

```diff
@@ -157,6 +157,12 @@ impl Commit {
         &self.data.committer
     }
 
+    ///  A commit is hidden, if its commit id is not in the predecessor set.
```

Comment:
nit: "not in the predecessor set" is not correct. Maybe "not in the change id index". (I think the comma should also be removed.)

Reply:
done